c++/src/kj/compat/lib.rs

```diff
@@ -0,0 +1,9 @@
+#[cxx::bridge(namespace = "kj")]
+mod gzip {
+    unsafe extern "C++" {
+        include!("kj/compat/gzip.h");
+
+        type GzipInputStream;
+        type GzipOutputStream;
+    }
+}
```

Comment:
Missing newline at end, `cargo fmt` would add that so probably other format errors too.